packages/framework/package.json

```diff
@@ -44,7 +44,8 @@
     "@deepkit/type": "^1.0.1-alpha.13",
     "@deepkit/workflow": "^1.0.1-alpha.13",
     "reflect-metadata": "^0.1.13",
-    "rxjs": "~6.6.0"
+    "rxjs": "~6.6.0",
+    "ws": "^7.4.2"
```

Comment:
We need `ws` package since we still use its types. And we have it as default implementation, so we should keep it installed per default to have an easier install experience.

Reply:
Oh wait, I thought for a second that you made it optional, but you didn't. Forget my comment about the types. However, we don't need ws in `peerDependencies`, `dependencies` is enough and easier.

Reply:
Hm, actually you may be right about the types as I use `import type` which optimizes runtime but `.d.ts` files would still reference those types.

I thought peerDependencies might be better if someone (like me :D) wants to use a different websocket implementation.  I will not need `ws` at runtime at all.